lib/api.js

```diff
@@ -0,0 +1,12 @@
+import Client from "./sanity";
+import { serialize } from "next-mdx-remote/serialize";
+
+export async function getAllTils() {
+	const results = await Client.fetch(`*[_type == "til"]{date, content}`);
+
+	for (let i = 0; i < results.length; i++) {
+		results[i].content = await serialize(results[i].content);
```

Comment:
I wonder, whether this would have worked, too:

```js
results = results.map(async (result) => await serialize(result.content))
```

Would require a `let`, though.

Reply:
<img width="945" alt="Screen Shot 2021-08-16 at 17 31 20" src="https://user-images.githubusercontent.com/33348046/129589317-42bdbc82-1463-4037-ac50-315e509c9eb1.png">

Reply:
Bummer :D

Thanks for checking.